cleverhans/attacks.py

```diff
@@ -25,16 +363,17 @@ def fgsm(x, predictions, eps, back='tf', clip_min=None, clip_max=None):
                     value for components of the example returned
     :return: a tensor for the adversarial example
     """
+    warnings.warn("attacks.fgsm is deprecated and will be removed on 09-27-17."
```

Comment:
I don't think we need to have a FastGradientSignMethod class. I just want to make sure that Google searches for "fgsm" find cleverhans code somewhere in the results page.